ruleset/ruleset_test.go

```diff
@@ -210,23 +208,29 @@ func TestGetRuleIdsToRun_OptionsAllDisabledAndConfigFileNil_returnAllStandardRul
 	}
 }
 
-func TestGetRuleIdsToRun_SpecificRuleHasInvalidRuleId_InvalidRuleIdError(t *testing.T) {
+func TestGetRuleIdsToRun_SpecificRuleHasInvalidRuleId_ReturnsInvalidRuleId(t *testing.T) {
 	//Given
 	opts := options.Options{
 		Rules: "InvalidId",
 	}
-	expectedStandardRuleIds := "Invalid rule ID: InvalidId"
+	expectedStandardRuleIds := []rules.RuleID{"InvalidId"}
 
 	//When
-	_, _, actualInvalidRuleIdError := GetRuleIdsToRun(nil, &opts)
+	actualStandardRuleIds, actualCustomRuleIds, err := GetRuleIdsToRun(nil, &opts)
 
 	//Then
-	if strings.TrimSpace(actualInvalidRuleIdError.Error()) != expectedStandardRuleIds {
-		t.Errorf("RuleIds mismatched")
+	if !reflect.DeepEqual(actualStandardRuleIds, expectedStandardRuleIds) {
+		t.Errorf("%s Actual: %+v, Expected: %+v", "Standard ruleIds are mismatched!", actualStandardRuleIds, expectedStandardRuleIds)
+	}
+	if !reflect.DeepEqual(len(actualCustomRuleIds), 0) {
+		t.Errorf("%s Actual: %+v, Expected: %+v", "Number of custom rule Ids should be 0!", len(actualCustomRuleIds), 0)
+	}
+	if err != nil {
+		t.Errorf("GetRuleIdsToRun method should not return error!")
```

Comment:
Can we also verify that it raised a warning at console regarding invalid ruleID?